src/Application.cpp

```diff
@@ -1,4 +1,4 @@
-#include "Application.h"
+﻿#include "Application.h"
```

Comment:
Well, this is weird. Looks like the line endings are being changed. Can you please fire up git bash, execute `git config --global core.autocrlf true`, save this line again, commit it and check if there are any changes?